app/assets/javascripts/utils/article_finder_utils.js

```diff
@@ -90,3 +90,30 @@ export const extractClassGrade = (pageAssessments) => {
   });
   return classGrade;
 };
+
+
+export const keywordAutocompleteGenerator = (keyword) => {
+  return {
+    list: 'search',
+    srsearch: keyword,
+    srlimit: 5,
+    srinfo: 'totalhits',
+    srprop: '',
+  };
+};
+
+
+const mediawikiApiBase = (language, project) => {
+  if (project === 'wikidata') {
+    return `https://${project}.org/w/api.php?action=query&format=json&origin=*`;
+  }
+  return `https://${language}.${project}.org/w/api.php?action=query&format=json&origin=*`;
+};
```

Comment:
There already exists a utility for this, best to use it https://github.com/WikiEducationFoundation/WikiEduDashboard/blob/15788d822067e2ef9da6c09a34f67a4946e056f2/app/assets/javascripts/utils/wiki_utils.js#L2-L8

Reply:
Thanks! I'll use that